components/training/eventBooking.tsx

```diff
@@ -167,6 +168,24 @@ const addRightBorder = (index) => {
   return (index + 1) % 3 != 0 ? "md:border-r-8" : "";
 };
 
+const EventDates = ({ duration, date }) => {
+  const startDate = dayjs(date);
+  const endDate = startDate.add(duration, "day");
+
+  const eventDateRange = () => (
```

Comment:
i'd recommend adding a comment to show what the expected output should be from this function

Reply:
Done